### DISCUSSION TITLE
MarkDown and Note Linking (Obsidian Style)

### DISCUSSION THREAD

**Discussion by Praczet** [hrsh7th/nvim-cmp#1737] [Q&A]
Hi,

I would like to know if there is a way to configure nvim-cmp to show list of files from root dir (Directory of current project)  after typing [[

To simulate the way as Obsidian works..

Best regards
Adam

**Comment by ben-hampson** [comment]
[obsidian.nvim](https://github.com/epwalsh/obsidian.nvim) offers this kind of auto-completion and utilises `nvim-cmp` to do it.

Check out the [configuration options](https://github.com/epwalsh/obsidian.nvim?tab=readme-ov-file#configuration-options), specifically the `completion` section. If you wanted you could probably disable all the other parts of `obsidian.nvim` and leave this part enabled.
